src/webui/webui.cpp

```diff
@@ -77,11 +79,18 @@ void WebUI::configure()
                 m_httpServer->close();
         }
 
-#ifndef QT_NO_OPENSSL
         if (pref->isWebUiHttpsEnabled()) {
-            const QByteArray certs = pref->getWebUiHttpsCertificate();
-            const QByteArray key = pref->getWebUiHttpsKey();
-            bool success = m_httpServer->setupHttps(certs, key);
+            const auto readData = [](const QString &path) -> QByteArray
+            {
+                QFile file(path);
+                if (!file.open(QIODevice::ReadOnly))
+                    return {};
+                return file.read(1024 * 1024);
```

Comment:
Please make a named constant at top of "webui.cpp".

Reply:
Are you sure? this is in a lambda function.

Reply:
Does it bother?
This number has a "configuration" value (aka MaxSSLFileSize). Now I see it used in several files so it should be in some shared place to prevent its independent changing.

Reply:
added Uilts::Net::MaxSSLFileSize